src/conn/conn_cache.c

```diff
@@ -46,6 +46,118 @@ __cache_config_abs_to_pct(
     return (0);
 }
 
+/*
+ * __validate_cache_config --
+ *     Validate trigger and target values of given configs.
+ */
+static int
+__validate_cache_config(WT_SESSION_IMPL *session, const char *cfg[], bool create)
+{
+    WT_CACHE *cache;
+    WT_CONFIG_ITEM cval;
+    WT_CONNECTION_IMPL *conn;
+    bool debug_config;
+
+    conn = S2C(session);
+    cache = conn->cache;
+
+    WT_RET(__wt_config_gets(session, cfg, "debug_mode.configuration", &cval));
+    debug_config = cval.val;
+
+    /* The target size must be lower than the trigger size or we will never get any work done. */
+    if (create) {
+        if (cache->eviction_target >= cache->eviction_trigger)
+            WT_RET_MSG(session, EINVAL, "eviction target must be lower than the eviction trigger");
```

Comment:
For each message, should we reuse the **exact same name** as the configuration? This way users will know what to change.
For instance:
```suggestion
            WT_RET_MSG(session, EINVAL, "eviction_target must be lower than the eviction_trigger");
```

Reply:
good point, I've modified the warning messages in: [feb593b](https://github.com/wiredtiger/wiredtiger/pull/10548/commits/feb593b0f3e88fc188dd0cc5c8b7725602d551f0)